src/clarity-angular/datagrid/datagrid-row.ts

```diff
@@ -20,6 +24,11 @@ import {Selection} from "./providers/selection";
     }
 })
 export class DatagridRow {
+
+    @ContentChildren(DatagridActionOverflow) private _actionableRows: QueryList<DatagridActionOverflow>;
+    public get actionableRows(): QueryList<DatagridActionOverflow> {
+        return this._actionableRows;
+    }
```

Comment:
Thanks for catching that. I actually don't think the second `*ngIf` is needed. I was checking for the wrong value there. It should be:
```
<clr-dg-cell *ngIf="rowActionService.actionableCount > 0" class="datagrid-single-select">
   <ng-content select="clr-dg-action-overflow"></ng-content>
</clr-dg-cell>
```
 This revised markup will always render the extra cell if any one of the rows have the extra cell for action. The ellipsis icon will only render if `<clr-dg-action-overflow>` exists in that row.